Server/controllers/auth.controller.js

```diff
@@ -1,13 +1,57 @@
 "use strict";
+const passport = require('passport');
+var jwt = require('jsonwebtoken');
+const passportConfig = require('../config/passport.config');
 /**
  * Class representing auth controller
  *
  * @class authController
  */
 class authController {
 
-    static login (req, res) {}
+  static login (req, res , next) {
+      passport.authenticate('local', function(err, user, info) {
+          if(err) {
+              return next(err);
+          }
+      
+          if (!user) { return res.json( { message: info.message }) }
+
+          const payload = {
+              username: user.username,
+              expires: Date.now() + parseInt(process.env.JWT_EXPIRATION_MS),
```

Comment:
What id `process.env.JWT_EXPIRATION_MS` is not defined?

Reply:
Please suggest what time should be defined and can be placed in default.json ?

Reply:
Set it to 24 hours

Reply:
@ashmeetkaur-compro please update this using value from config

Reply:
Updated expiry to 24 hrs